cpp/src/cider/type/plan/Expr.h

```diff
@@ -200,6 +200,8 @@ class Expr : public std::enable_shared_from_this<Expr> {
 
   bool isAutoVectorizable() const { return auto_vectorizable_; }
 
+  bool isColumnVar() const { return is_column_var_; }
```

Comment:
we always check type by dynamic_cast, why add this flag?

Reply:
Reduce RTTI cost.